src/providers/FusionAuthProvider.tsx

```diff
@@ -97,21 +105,47 @@ export const FusionAuthProvider: React.FC<Props> = ({ config, children }) => {
         [config, generateUrl],
     );
 
+    useEffect(() => {
+        const userCookie = Cookies.get('user');
+        if (userCookie) {
+            setUser(JSON.parse(userCookie));
+        }
+    }, [setUser]);
+
+    const refreshToken = useCallback(async () => {
+        fetch(`${config.serverUrl}/jwt-refresh`, {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/json',
+            },
+            credentials: 'include',
+        });
+    }, []);
```

Comment:
Can we add `serverUrl` to dependencies here? Also I think we will have to refactor this so the properties are not a config object. Otherwise the users of the SDK would have to memoize the config object which is not ideal usage